Updated Login Page to add new refresh token pointer (#3982)
MicrosoftDocs/azure-docs-cli: docs-ref-conceptual/msal-based-azure-cli.md

### Calling `az account get-access-token`

You can manually call [`az account get-access-token`](/cli/azure/account#az_account_get_access_token) in a terminal or use a subprocess to call it from another programming language. By default, the returned access token is for Azure Resource Manager (ARM) and the default subscription/tenant shown in [`az account show`](/cli/azure/account#az_account_show). 

> Starting from Azure CLI 2.54.0, `az account get-access-token` returns an additional property which makes it supported by MSAL-based Azure CLI. See the following for an example of the access token output of the changes:

Reviewer: This sentence about access tokens is inside a code block to get the active subscription?  Maybe GitHub preview is a little confused.

```azurecli-interactive

# Verify the active subscription.
az account show --output table

# Get access token for the active subscription.
az account get-access-token

```

> Console output:

Reviewer: Put the new /Include file above this line?

### Calling `az account get-access-token`

You can manually call [`az account get-access-token`](/cli/azure/account#az_account_get_access_token) in a terminal or use a subprocess to call it from another programming language. By default, the returned access token is for Azure Resource Manager (ARM) and the default subscription/tenant shown in [`az account show`](/cli/azure/account#az_account_show). 

Starting from Azure CLI 2.54.0, `az account get-access-token` returns an additional property which makes it supported by MSAL-based Azure CLI. See the following for an example of the access token output of the changes:

> Output Console:

Reviewer: ```suggestion
Console output:
```

Reviewer: Line 47 needs "```" but if I put it in, GitHub doesn't save the suggestion correctly.

```output
# Before:

{
  "accessToken": "...",
  "expiresOn": "2023-10-31 21:59:10.000000",
  "subscription": "...",
  "tenant": "...",
  "tokenType": "Bearer"
}

# After:

{
  "accessToken": "...",
  "expiresOn": "2023-10-31 21:59:10.000000",
  "expires_on": 1698760750,
  "subscription": "...",
  "tenant": "...",
  "tokenType": "Bearer"
}
```

> To learn more in-depth about the changes presented in this access token output example, see [Manage Azure Subscriptions for Azure CLI](./manage-azure-subscriptions-azure-cli.md#access-tokens). 

Reviewer: ```suggestion
To learn more about the access tokens, see [Sign in with Azure CLI](./authenticate-azure-cli#refresh-tokens). 
```
